hasher-matcher-actioner/terraform/terraform.tfvars.example

```diff
@@ -7,3 +7,6 @@ log_retention_in_days = # DAYS to retain logs
 include_cloudfront_distribution = true
 te_api_token = "<threat-exchange-api-token>"
 collab_file = "collabs_example.json"
+use_shared_user_pool = false # if true, run authentication-shared terraform, and use its outputs for webapp_and_api_shared_user_pool_id and webapp_and_api_shared_user_pool_client_id
+webapp_and_api_shared_user_pool_id = "us-east-1_T11ZBU6gM"
```

Comment:
did you mean to hardcode these are they at all sensitive?

Reply:
I think I thought about it and thought why not leave them. They're randomly generated and not in use by us... but others haven't been left in that way so I'll change.

Reply:
Yeah, I think it probably makes sense to remove because of like you said, not known what is in use vs an example, if you wanted to add a comment on that line with an "e.g. ..." that could work.